multiplayer/src/index.tsx

```diff
@@ -49,7 +49,7 @@ window.addEventListener("DOMContentLoaded", () => {
         const wc = pxt.webConfig as any;
 
         for (const key of Object.keys(wc)) {
-            if (wc[key]?.startsWith("/") && wc[key]?.indexOf("worker") == -1) {
+            if (typeof wc[key] === "string" && wc[key]?.startsWith("/") && wc[key]?.indexOf("worker") == -1) {
```

Comment:
Hmm, this makes me a bit worried that we have this constraint other places.. I'll have to take a look around to make sure I'm not breaking anything else. Thanks for fixing this here.

Reply:
No worries. TBH I don't think ocv is at fault here. Just an unsafe assumption in the code. ocv wasn't actually the first boolean value added to the config, just the first one that was actually set at this point.